From fc2c56a52421a68ac41e5162da03eb909440b169 Mon Sep 17 00:00:00 2001
From: Thibault Saunier <thibault.saunier@osg.samsung.com>
Date: Thu, 3 Aug 2017 17:03:31 -0400
Subject: [PATCH 2/2] transition: Fix usage of transition types != crossfade

We need to make sure the crossfade ratio is disabled in that case.
---
 ges/ges-video-transition.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ges/ges-video-transition.c b/ges/ges-video-transition.c
index 42906201..29479c5a 100644
--- a/ges/ges-video-transition.c
+++ b/ges/ges-video-transition.c
@@ -465,7 +465,7 @@ ges_video_transition_update_control_sources (GESVideoTransition * self,
         duration, 0.0, 0.0);
   } else {
     ges_video_transition_update_control_source
-        (priv->crossfade_control_source, duration, 1.0, 1.0);
+        (priv->crossfade_control_source, duration, -1.0, -1.0);
     ges_video_transition_update_control_source (priv->smpte_control_source,
         duration, 1.0, 0.0);
   }
-- 
2.13.3

